packages/client/lib/execution/vmexecution.ts

```diff
@@ -49,7 +50,7 @@ export class VMExecution extends Execution {
 
     if (this.config.vm === undefined) {
       const trie = new Trie({
-        db: new LevelDB(this.stateDB),
+        db: new LevelDB(this.stateDB) as DB<Uint8Array, Uint8Array>,
```

Comment:
```suggestion
        db: new LevelDB(this.stateDB) as DB,
```
I don't think the types are necessary since this defaults to Uint8Array.